src/actions/guides/database/querying.cr

```diff
@@ -482,7 +498,7 @@ class Guides::Database::Querying < GuideAction
     ```
 
     > Since associations take a block, you can also use [Short one-argument syntax](https://crystal-lang.org/reference/syntax_and_semantics/blocks_and_procs.html#short-one-argument-syntax).
-    > (e.g. `tasks(&.completed(true).updated_at.get(1.day.ago))`)
+    > (e.g. `where_tasks(&.completed(true).updated_at.get(1.day.ago))`)
```

Comment:
Same here with the query  object

Reply:
actually that whole block of text just goes away now.

Reply:
Oh yeah :D